### PR TITLE
rewrite error messages as full sentences

### LINKED ISSUE #37: Rewrite error messages as full sentences
cf https://github.com/dobtco/screendoor-v2/issues/1550#issuecomment-78840660

### PR DESCRIPTION
**Pull request by ajb**
closes #37 

```
FormRenderer.errors =
  blank: "This field can't be blank."
  invalid_date: 'Please enter a valid date.'
  invalid_email: 'Please enter a valid email address.'
  invalid_integer: 'Please enter an even integer.'
  invalid_number: 'Please enter a valid number.'
  invalid_price: 'Please enter a valid price.'
  invalid_time: 'Please enter a valid time.'
  too_large: 'Your entry is too large.'
  too_long: 'Your entry is too long.'
  too_short: 'Your entry is too short.'
  too_small: 'Your entry is too small.'
```

I kinda freestyled it with some of the copy. @jrubenoff, thoughts?

### CODE REVIEW

File: dist/formrenderer.standalone.uncompressed.js [hunk 1]
```diff
@@ -571,6 +571,20 @@
 
   FormRenderer.ORDERED_COUNTRIES = _.uniq(_.union(commonCountries, [void 0], _.keys(ISOCountryNames)));
 
+  FormRenderer.errors = {
+    blank: "This field can't be blank.",
+    invalid_date: 'Please enter a valid date.',
+    invalid_email: 'Please enter a valid email address.',
+    invalid_integer: 'Please enter an even integer.',
```
Comment (jrubenoff) [inline dist/formrenderer.standalone.uncompressed.js]: Maybe 'Please enter a whole number'?

File: dist/formrenderer.standalone.uncompressed.js [hunk 2]
```diff
@@ -571,6 +571,20 @@
...
+    invalid_number: 'Please enter a valid number.',
+    invalid_price: 'Please enter a valid price.',
+    invalid_time: 'Please enter a valid time.',
+    too_large: 'Your entry is too large.',
+    too_long: 'Your entry is too long.',
```
Comment (jrubenoff) [inline dist/formrenderer.standalone.uncompressed.js]: What do you think about changing 'entry' to 'answer'?

**Comment by jrubenoff** [comment]
Commented. Looks good otherwise!

1. What will still remain unchanged in Screendoor after this is merged?
2. What about the "short" versions of each error?
3. Might already know the answer to this, but is there an error for file uploads which exceed our size limits?

**Comment by ajb** [comment]
1. If we're just talking about response validation (and not any other form validation), the only thing that will be inconsistent is the unsubmitted responses page.
2. That's not part of formrenderer, so it's not part of this PR.
3. Nope

File: dist/formrenderer.standalone.uncompressed.js (hunk 2 above)
Comment (ajb) [inline dist/formrenderer.standalone.uncompressed.js]: :+1:

File: dist/formrenderer.standalone.uncompressed.js (hunk 1 above)
Comment (ajb) [inline dist/formrenderer.standalone.uncompressed.js]: :+1:

**Comment by ajb** [comment]
I should clarify: the file upload errors are generated by the server, and formrenderer-base just displays whatever it receives in the AJAX response.

**Pull request merged**